otx/algorithms/detection/tasks/task.py

```diff
@@ -0,0 +1,608 @@
+"""Task of OTX Detection."""
```

Comment:
Do we need to use `tasks` folder in task folder (i.e. `detection`)?
If this `detection/tasks` folder just includes only one file (`task.py`), how about bring `task.py` to upper level? (i.e. `otx/algorithms/detection/task.py`)

Reply:
Maybe nncf.py and openvino.py will remain for sometime(or forever?) How about it?

Reply:
After talking with Wonju, we're going to move nncf.py and openvino.py to adapter level, then we can delete tasks folder and move task.py to parent directory